backend/engine/plugins/trivy_sbom/parser.py

```diff
@@ -19,7 +19,10 @@ def clean_output_application_sbom(output: list) -> list:
         if item["type"] == "operating-system":
             continue
         bom_ref = item["bom-ref"]
-        name = item.get("group", "") + item["name"]
+        if "group" in item:
+            name = f'{item["group"]}/{item["name"]}'
+        else:
+            name = item["name"]
```

Comment:
should we be `None` type safe here and just make it name = item.get("name") instead?

Reply:
If the name of the package is missing, that is a bigger issue. I'll add a check to handle a missing `name` field because other components depend on that field.

FWIW: The cyclone dx schema mentions that `name` is a required field - https://github.com/CycloneDX/specification/blob/1.6/schema/bom-1.6.schema.json#L830-L832 

But i will add a check incase its missing in the trivy result